README.adoc

```diff
@@ -18,7 +17,24 @@ variable "ENV" to "inmemory" and do a rebuild. The setting is stored inside the
 changing the value back to unset or "development" (default for npm start) needs a rebuild
 to take effect.
 
-== Production build
+== Functional Tests
+
+Functional (UI) tests are automatically executed on the PR check process (using 
+`cico_run_tests.sh`) or on the release process (using `cico_build_deploy.sh`) but
+can also executed from the development environment by invoking the following command:
+
+----
+./run_functional_tests.sh
+----
+
+External resources are mocked. A Java JRE, ncat and fuser have to be available on the
+system (see Dockerfile.builder for details on the required packages when building on
+Fedora/CentOS). PhantomJS will be used as the browser engine by default. This can be
```

Comment:
Running it from the container does not need any new instructions, it ist already integrated in the existing process, so the instructions in the Readme still apply and will invoke the tests. In fact, I just replaced the "Some tests here...?" comment in the scripts with the execution of the tests.